python/paddle/v2/fluid/tests/book/test_understand_sentiment_lstm.py

```diff
@@ -1,160 +0,0 @@
-#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
This network uses the static RNN, which is deprecated and not supported in Fluid majorly.